test/CMakeLists.txt

```diff
@@ -46,3 +46,44 @@ foreach(GTEST_SOURCE_file ${tests})
 
    set_tests_properties(${BINARY_NAME} PROPERTIES TIMEOUT 240)
 endforeach()
+
+find_package(PythonInterp)
+
+if(NOT PYTHONINTERP_FOUND)
+  message(STATUS "No PythonInterp found. Linters will not be executed")
+  return()
+endif()
+
+find_program(WGET_EXE wget)
+if(WGET_EXE)
+  message(STATUS "Found WGet: ${WGET_EXE}")
+  add_custom_target(wget_cppcheck
+                    COMMAND ${WGET_EXE} -q -O cppcheck.py https://raw.githubusercontent.com/ament/ament_lint/master/ament_cppcheck/ament_cppcheck/main.py
```

Comment:
Does the test work only by downloading the `main.py` file?

Reply:
yes, it's enough